src/Microsoft.Health.Dicom.Blob/Features/Export/AzureBlobExportSinkProvider.cs

```diff
@@ -71,8 +80,8 @@ protected override async Task<IExportSink> CreateAsync(IServiceProvider provider
         return new AzureBlobExportSink(
             provider.GetRequiredService<IFileStore>(),
             await options.GetBlobContainerClientAsync(
-                provider.GetRequiredService<IExportIdentityProvider>(),
-                provider.GetRequiredService<IOptionsMonitor<AzureBlobClientOptions>>().Get("Export"),
+                provider.GetRequiredService<IServerCredentialProvider>(),
+                provider.GetRequiredService<IOptionsSnapshot<AzureBlobClientOptions>>().Get("Export"),
```

Comment:
We resolve this per scope

Reply:
Yes, and this is a good point. Right now there isn't too much of a reason why I retrieve some resources via the explicit `IServiceProvider` while others I fetch via the ctor. I can normalize this